idunn/places/poi.py

```diff
@@ -0,0 +1,68 @@
+from .place import Place
+from idunn.api.utils import build_blocks, get_geom, get_name
+
+
+class POI(Place):
+    PLACE_TYPE = 'poi'
+
+    @classmethod
+    def build_address(cls, es_place):
```

Comment:
I don't understand, why a different implementation is needed, compared to `Place.build_address`